Tactility/Source/app/usbsettings/UsbSettings.cpp

```diff
@@ -11,32 +11,46 @@
 
 namespace tt::app::usbsettings {
 
-static void onRebootMassStorage(TT_UNUSED lv_event_t* event) {
+static void onRebootMassStorageSdmmc(TT_UNUSED lv_event_t* event) {
     hal::usb::rebootIntoMassStorageSdmmc();
 }
 
+// Flash reboot handler
+static void onRebootMassStorageFlash(TT_UNUSED lv_event_t* event) {
+    hal::usb::rebootIntoMassStorageFlash();
+}
+
 class UsbSettingsApp : public App {
 
     void onShow(AppContext& app, lv_obj_t* parent) override {
         auto* toolbar = lvgl::toolbar_create(parent, app);
         lv_obj_align(toolbar, LV_ALIGN_TOP_MID, 0, 0);
 
         if (hal::usb::canRebootIntoMassStorageSdmmc()) {
-            auto* button = lv_button_create(parent);
-            auto* label = lv_label_create(button);
-            lv_label_set_text(label, "Reboot as USB storage");
-            lv_obj_align(button, LV_ALIGN_CENTER, 0, 0);
-            lv_obj_add_event_cb(button, onRebootMassStorage, LV_EVENT_SHORT_CLICKED, nullptr);
-        } else {
+            // Existing SDMMC button
+            auto* button_sd = lv_button_create(parent);
+            auto* label_sd = lv_label_create(button_sd);
+            lv_label_set_text(label_sd, "Reboot as USB storage (SD)");
+            lv_obj_align(button_sd, LV_ALIGN_CENTER, 0, -20);  // Above flash button
```

Comment:
`-20` wont' work when buttons scale up in size, and also not super well when they scale down a lot.

What I'd do:
1. Make a lv_obj as a wrapper, and align in to center of the screen.
2. Add these widgets to it and center them horizontally